src/main/java/com/hivemq/extension/sdk/api/client/parameter/ConnectionAttributeStore.java

```diff
@@ -111,27 +115,33 @@
      * @param key     The key of the connection attribute.
      * @param charset The {@link Charset} of the value of the connection attribute.
      * @return An {@link Optional} containing the value of the connection attribute as a string with the given charset
-     *         if present.
+     * if present.
      * @since 4.0.0, CE 2019.1
      */
     @NotNull Optional<String> getAsString(@NotNull String key, @NotNull Charset charset);
 
     /**
      * Retrieves all connection attributes for the connected client.
      *
```

Comment:
missing <p>, please enable automatic formatting, as we have now a proper code style config (at least in this repo)

Reply:
This is all auto formated

Reply:
No, it's not, just tested it out